loader.py

```diff
@@ -0,0 +1,38 @@
+"""
+Shaurya Pratap Singh 
+@shaurya-blip
+
+Shows loading message while doing something.
+"""
+
+import itertools
+import threading
+import time
+import sys
+
+# The task is not done right now
+done = False
+
+
+def animate(message="loading", endmessage="Done!"):
```

Comment:
For smaller file size I would replace message with start and endmessage with end